packages/transaction-messages/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "compilerOptions": {
+        "lib": ["DOM", "ES2017", "ES2022.Error"]
```

Comment:
Check if you need all of these.

Reply:
Removed them all for now, I'll add whatever's needed as I go